pkg/apis/cluster/v1alpha1/types.go

```diff
@@ -127,6 +127,9 @@ type LocalSecretReference struct {
 const (
 	// ClusterConditionReady means the cluster is healthy and ready to accept workloads.
 	ClusterConditionReady = "Ready"
+	// ClusterConditionWarn means some err occurs when access member cluster but it doesn't affect the use of member cluster
+	// ClusterConditionWarn are ready to accept workloads too
+	ClusterConditionWarn = "Warn"
```

Comment:
`Warn` seems too common for me.
Let's try to work out a better one.

Reply:
What is your suggestion?How about `alarm`,I am limited by my english level.

Reply:
> `APIEnable`?

`warn `means control plane can not get all api list in member cluster now,perhaps a condition names `APIUnreachable` ?

Reply:
@Tingtal 
Need your help here.

The Karmada system will gather some information from a cluster, such as `version`, `apis`, etc. 
We want to define a `state` to describe if all required information is collected. 

Assume the `state` is `A`:
- A= true means all required information is collected.
- B= false means some of the required information is missed.

Reply:
Post an option(brainstorming):
`positive`: 
- `true` means we have enough confidence with the cluster based on sufficient information we got.
- `false` means lack of confidence as we cant know the cluster clearly as missing information.

Reply:
another option:
`MetadataSufficient`: 
- `true`: We get all the required data
- `false`: We didn't get all the required data, some of them are missing.

Reply:
Ideas to share:

dataSufficient
dataIntact
requiredDataObtained
requiredDataReady
dataAllObtained
dataAllReady
inPlace

Reply:
`MetadataSufficient `+1

Reply:
How about talking about this and deciding at the next community meeting? @duanmengkk 

More work to do after the condition I guess.

Reply:
ok,let's discuss it

Reply:
Fine. thanks. please remember to add an agenda to the meeting notes.

Reply:
The latest code judge the health status of the cluster by `/readyz` and `/healthz` and ignore the error when `get pod` and `get node`,`get api-resource` ,just log the error and do not fail over.
So,should I close the PR ? @RainbowMango
![image](https://user-images.githubusercontent.com/17697155/170056998-3c58eedb-d904-4f71-842a-ebf2088429ff.png)